lambdas/question/src/main/java/uk.gov.di.ipv.cri.kbv.api.handler/QuestionHandler.java

```diff
@@ -47,25 +52,26 @@
     private final ObjectMapper objectMapper;
     private final KBVStorageService kbvStorageService;
     private final PersonIdentityService personIdentityService;
-
+    private final EventProbe eventProbe;
+    private final KBVService kbvService;
+    private final AuditService auditService;
     private final ConfigurationService configurationService;
-
     private final Clock clock;
-    private APIGatewayProxyResponseEvent response;
-    private EventProbe eventProbe;
-    private KBVService kbvService;
 
     @ExcludeFromGeneratedCoverageReport
     public QuestionHandler() {
-        this.objectMapper = new ObjectMapper();
-        this.objectMapper.registerModule(new JavaTimeModule());
+        this.objectMapper = new ObjectMapper().registerModule(new JavaTimeModule());
         this.kbvStorageService = new KBVStorageService();
         this.personIdentityService = new PersonIdentityService();
         this.kbvService = new KBVServiceFactory().create();
         this.configurationService = new ConfigurationService();
 
-        this.response = new APIGatewayProxyResponseEvent();
         this.eventProbe = new EventProbe();
+        this.auditService =
+                new AuditService(
+                        SqsClient.builder().build(),
+                        new ConfigurationService(),
+                        new ObjectMapper());
```

Comment:
change this line to refer to this.objectMapper

Reply:
Done.